src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="style.css"
+      method="post"
+      data-qa="big"
+      class="bigForm"
```

Comment:
Due to the task checklist you have to create general class for example `form ` and move all similar styles to it and add this class to your big and small forms, also for your small form you need to create a modifier to add additional styles
![image](https://user-images.githubusercontent.com/80700247/229276033-1f706a1b-6cd9-49a7-952c-bfd4af3bb906.png)

Fix everywhere